modules/govuk/manifests/apps/content_data_admin.pp

```diff
@@ -0,0 +1,103 @@
+# == Class: govuk::apps::content_data_admin
+#
+# Read more: https://github.com/alphagov/content_data_admin
+#
+# === Parameters
+# [*port*]
+#   What port should the app run on? Find the next free one in development-vm/Procfile
+#
+# [*enabled*]
+#   Whether to install or uninstall the app. Defaults to true (install on all enviroments)
+#
+# [*secret_key_base*]
+#   The key for Rails to use when signing/encrypting sessions (in govuk-secrets)
+#
+# [*sentry_dsn*]
+#   The app-specific URL used by Sentry to report exceptions (in govuk-secrets)
+#
+# [*oauth_id*]
+#   The OAuth ID used to identify the app to GOV.UK Signon (in govuk-secrets)
+#
+# [*oauth_secret*]
+#   The OAuth secret used to authenticate the app to GOV.UK Signon (in govuk-secrets)
+#
+# [*db_hostname*]
+#   The hostname of the database server to use for in DATABASE_URL environment variable
+#
+# [*db_username*]
+#   The username to use for the DATABASE_URL environment variable
+#
+# [*db_password*]
+#   The password to use for the DATABASE_URL environment variable
+#
+# [*db_port*]
+#   The port of the database server to use in the DATABASE_URL.
+#   Default: undef
+#
+# [*db_name*]
+#   The database name to use for the DATABASE_URL environment variable
+#
+class govuk::apps::content_data_admin (
+  $port = '3230',
+  $enabled = false,
+  $secret_key_base = undef,
+  $sentry_dsn = undef,
+  $oauth_id = undef,
+  $oauth_secret = undef,
+  $db_username = 'content_data_admin',
+  $db_hostname = undef,
+  $db_port = undef,
+  $db_allow_prepared_statements = undef,
+  $db_password = undef,
+  $db_name = 'content_data_admin_production',
+) {
+  $app_name = 'content-data-admin'
+
+  $ensure = $enabled ? {
+    true  => 'present',
+    false => 'absent',
+  }
+
+  # see modules/govuk/manifests/app.pp for more options
+  if $enabled {
+    govuk::app { $app_name:
+      ensure            => $ensure,
```

Comment:
As `$ensure` is only set to `present` if `$enabled` is `true` then do we need the `if $enabled {` block on line 62?

Reply:
Doesn't look like it according to this: https://github.com/alphagov/govuk-puppet/pull/7762/files Cc @KyleMacPherson @peteglondon

Reply:
This was the initial advise from 2nd line early on Wednesday. We've since gained a bit more understanding about puppet so you are correct, this is not required.

We'll get rid of it now